uint32 sequenceNr
Header header
geometry_msgs/Point position
geometry_msgs/Twist velocity
float64 air_speed
float64 ground_speed
geometry_msgs/Accel acceleration
geometry_msgs/Vector3 attitude
float64 angle_of_attack
float64 angle_of_sideslip
bool gps_fix
bool autopilot_active
float64 battery_voltage
float64 battery_current
float64 motor_speed
float64 aileron_level
float64 elevator_level
float64 rudder_level
float64 throttle_level
float64 precipitation_level
float64 storminess_level
wind_layer[] wind_layers